### PR TITLE
[Unit Test] Show source file when debugging

### PR DESCRIPTION
**Pull request by ketttY**
Fixes: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1000552

### CODE REVIEW

File: main/src/addins/MonoDevelop.UnitTesting/Gui/TestPad.cs [hunk 1]
```diff
@@ -450,11 +450,25 @@ protected void OnUpdateRunTest (CommandArrayInfo info)
 		[CommandHandler (TestCommands.DebugTest)]
 		protected void OnDebugTest (object data)
 		{
+			openSourceFile ();
 			var debugModeSet = Runtime.ProcessService.GetDebugExecutionMode ();
```
[the remark this answers is not in the document]
Comment (mrward) [reply main/src/addins/MonoDevelop.UnitTesting/Gui/TestPad.cs]: Not sure I can see anything obvious that would have changed this between 8.3.1 and 8.3.2 looking at the diff:

https://github.com/mono/monodevelop/compare/monodevelop-8.3.1.18...monodevelop-8.3.2.32

[the remark this answers is not in the document]
Comment (mrward) [reply main/src/addins/MonoDevelop.UnitTesting/Gui/TestPad.cs]: Yeah this is definitely working in 8.3.1 but not working in 8.3.2. Although it is not obvious to me what has changed here. 😕

[the remark this answers is not in the document]
Comment (mrward) [reply main/src/addins/MonoDevelop.UnitTesting/Gui/TestPad.cs]: VS Mac 8.3.1 is opening Test.cs for me twice if it was closed when the assert failed. Which is slightly wrong.

**Comment by ketttY** [comment]
Seems like someone fixed it? 
openfile

**Comment by mrward** [comment]
Not working for me in VS Mac 8.4.0.1971 using an NUnit library project nor with an xUnit .NET Core test project.

**Pull request closed without merging**